modules/util/source-formatter/src/main/java/com/liferay/source/formatter/upgrade/GradleBuildFile.java

```diff
@@ -161,7 +161,7 @@ public void insertGradleDependency(
 	}
 
 	private void _saveSource(List<String> lines) {
-		_source = StringUtil.merge(lines, System.lineSeparator());
+		_source = StringUtil.merge(lines, System.lineSeparator(), false);
```

Comment:
@slnn Oh, I didn't notice that `StringUtil.merge` will `trim()` the strings. Instead of adding new method to `StringUtil`, you can just use raw `StringBundler` here -- make sure you claim it clearly in the commit message.

Reply:
@slnn By the way, it seems the `StringUtil.merge` from petra-string doesn't auto trim.